server/handlers/ugv/dummy.py

```diff
@@ -69,7 +74,7 @@ def readmission(filename):
 class DummyUGVHandler:
     def __init__(self, gs, config):
         self.logger = logging.getLogger("groundstation")
-        self.gs = gs
+        self.gs: GroundStation = gs
```

Comment:
I don't really care that much, but if you want you could put all these type annotations and small other stuff into a separate code cleanup commit.